packages/canvas-panel-cookbook/src/pages/SlideShow/components/SimpleSlideTransition.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import './SimpleSlideTransition.scss';
+
+let cnt = 0;
+const SimpleSlideTransition = props => {
+  let { children } = props;
+  return (
+    <TransitionGroup className="slide-transitions">
+      <CSSTransition key={cnt++} timeout={500} classNames="fade">
```

Comment:
The `cnt` variable here will force CSS transition to re-render every time. If they key changes, the whole component will be unmounted and then remounted. Is that expected behaviour? We could use `shouldComponentUpdate` and always return false, which IMO has clearer intent. If not, maybe we can pass a key down from the props.